doc/python/static-image-export.md

```diff
@@ -208,31 +188,80 @@ And, here is an example of specifying that Kaleido should be used:
+## plotly.io Functions
 
-```python
+Previous examples on this page access `write_image` and `to_image` as methods on Plotly Figure objects. This functionality is also available via the `plotly.io` subpackage.
+
+The following example uses the `write_image` function from  `plotly.io`. The function takes the figure or a `dict` representing a figure (as shown in the example) as it's first argument.
+
+
+~~~python
 import plotly.io as pio
-pio.kaleido.scope.default_format = "svg"
-```
 
-Here is a complete listing of the available image export settings:
 
- - **`default_width`**: The default pixel width to use on image export.
- - **`default_height`**: The default pixel height to use on image export.
- - **`default_scale`**: The default image scale factor applied on image export.
- - **`default_format`**: The default image format used on export. One of `"png"`, `"jpeg"`, `"webp"`, `"svg"`, `"pdf"`, or `"eps"`.
- - **`mathjax`**: Location of the MathJax bundle needed to render LaTeX characters. Defaults to a CDN location. If fully offline export is required, set this to a local MathJax bundle.
- - **`topojson`**: Location of the topojson files needed to render choropleth traces. Defaults to a CDN location. If fully offline export is required, set this to a local directory containing the [Plotly.js topojson files](https://github.com/plotly/plotly.js/tree/master/dist/topojson).
- - **`mapbox_access_token`**: The default Mapbox access token.
+fig = dict({
+    "data": [{"type": "bar",
+              "x": [1, 2, 3],
+              "y": [1, 3, 2]}],
+    "layout": {"title": {"text": "A Figure Specified By Python Dictionary"}}
+})
 
+pio.write_image(fig, "fig.png")
+~~~
 <!-- #endregion -->
 
-### Image Export Settings (Orca)
-See the [Orca Management](/python/orca-management/) section for information on how to specify image export settings when using orca.
+## Image Export Settings (Kaleido)
+
+As well as configuring height, width, and other settings by passing arguments when calling `write_image` and `to_image`, you can specify defaults to be used. 
+
+### Available Settings
+
+The following settings are availble. 
+
+`default_width`: The default pixel width to use on image export.
+
+`default_height`: The default pixel height to use on image export.
+
+`default_scale`: The default image scale factor applied on image export.
+
+`default_format`: The default image format used on export. One of "png", "jpeg", "webp", "svg", "pdf", or "eps" (Kaleido v0 only).
+
+`mathjax`: Location of the MathJax bundle needed to render LaTeX characters. Defaults to a CDN location. If fully offline export is required, set this to a local MathJax bundle.
+
+`topojson`: Location of the topojson files needed to render choropleth traces. Defaults to a CDN location. If fully offline export is required, set this to a local directory containing the Plotly.js topojson files.
+
```

Comment:
TODO: Confirm status of `mathjax` and `topojson` before full release and adjust docs.

Reply:
what was the resolution of the TODO?

Reply:
Those settings were added here https://github.com/plotly/plotly.py/pull/5062/files#diff-3286c2428b898cb1f89f41c8a58258c84d68aa65fe4a60c13cfa6a8be3ddf399R14-R15
since I wrote the todo.